config.def.h

```diff
@@ -67,6 +67,9 @@ static const int THUMB_SIZE = 3;
 #endif
 #ifdef _MAPPINGS_CONFIG
 
+/* Bitmask of modifiers whose presence in key events/keybindings are ignored */
+static const int ignore_mask = Mod2Mask | LockMask;
```

Comment:
Before, if caps-lock was turned on and user pressed `a` , that would get treated as `a` or `A` ?
And what is the current behavior with the changes?

Reply:
I'm pretty confident the behavior hasn't changed. Believe "a" would be treated as "A" both before and now

Reply:
Can you test it and confirm? I'd do it myself but I don't have capslock.

Reply:
Neither do I. (Does anyone actually use capslock?). But I'm very confident that the behavior of capslock hasn't changed due to this commit

Reply:
i don't have a capslock really but got right shift to emulate capslock when tapped and be shift when held
| patch | <kbd>CapsLock</kbd> on |
|----|----|
| before | A |
| after | A |

so no it doesn't change the previous behaviour of <kbd>a</kbd> being treated as A when <kbd>CapsLock</kbd> is on